composer.json

```diff
@@ -86,5 +86,10 @@
             "**/tests/",
             "**/Test/"
         ]
+    },
+    "autoload-dev": {
```

Comment:
why did you add this one?
Is it a new namespace?

Reply:
Tests in Symfony 4 are in `/tests/`.
So I already provided this in the composer.json and placed the tests in the right position.
The namespace `ForkCMS\Tests` was not already existing, so that is probably OK then.